json-definitions/v1/test-result/index.json

```diff
@@ -278,6 +278,13 @@
     },
     "deletionFlag": {
       "type": "boolean"
+    },
+    "recalls": {
```

Comment:
This should be any of ref or null I think, because you shouldn't need to send it

Reply:
Hmm is that true.... this could be left off as not required so ignore that, but should recalls:null be allowed? my guess is yes because of how VTM does stuff

Reply:
To your first comment, in the type it is optional:

`recalls?: RecallsSchema;`

With regards to recalls being null, it's an optional key, so I guess what would what would be the value in allowing them to send it with a null value? If it's null, why include it in the payload?

Reply:
Or does VTM always send it as part of their mapping, and then just assign null?

Reply:
I guess if brokering API returned 500, then they would not have anything assigned as the value in the /recalls response, so in that case could be null?